README.md

```diff
@@ -877,6 +878,65 @@ Data type: `String`
 
 Name. Made up of access_list and the entry with an underscore seperator. eg. list42_10 is from access_list list42 and entry 10.
 
+#### ios_network_trunk
+
+Ethernet logical (switch-port) interface.  Configures VLAN trunking. Extension of network_trunk.
```

Comment:
as with the `ios_radius_global` work, the docs should show only the new attributes and link back to `network_trunk`

Reply:
Done